src/commonjs.cts

```diff
@@ -0,0 +1,9 @@
+type RequireCallback = (err: Error | undefined, lyra?: unknown) => void;
```

Comment:
is `cts` a valid extension?!

Reply:
Apparently yes :)

https://www.typescriptlang.org/docs/handbook/esm-node.html

Reply:
Yes it is, quite unknown tho